app/packages/state/src/hooks/useSchemaSettings.ts

```diff
@@ -26,23 +33,28 @@ export const TAB_OPTIONS_MAP = {
 
 export const TAB_OPTIONS = Object.values(TAB_OPTIONS_MAP);
 
-const skipFiled = (path: string, ftype: string) => {
+const skipFiled = (path: string, ftype: string, schema: {}) => {
+  const parentPath = path.substring(0, path.lastIndexOf("."));
+
   return (
-    ftype === DICT_FIELD ||
+    UNSUPPORTED_FILTER_TYPES.includes(path) ||
     ftype === JUST_FIELD ||
-    path.includes(".logits") ||
-    path.endsWith(".index") ||
-    path.endsWith(".bounding_box")
+    (parentPath &&
+      schema[parentPath]?.embeddedDocType === DETECTION_FILED &&
+      path.endsWith(".bounding_box")) ||
+    path.endsWith(".index")
   );
 };
-const disabledField = (path: string, ftype: string, subfield: string) => {
+const disabledField = (path: string, ftype: string, groupField?: string) => {
   return (
     path === "tags" ||
     path === "filepath" ||
     (ftype === OBJECT_ID_FIELD && path === "id") ||
+    ftype === FRAME_NUMBER_FIELD ||
+    groupField === path ||
+    path === "sample_id" ||
     path.startsWith("metadata") ||
-    path.endsWith("frames.frame_number") ||
-    ftype === FRAME_NUMBER_FIELD
+    path.endsWith("frames.frame_number")
```

Comment:
Using the [`FrameNumberField`](https://github.com/voxel51/fiftyone/blob/c9f37ad3f8ea52bb69d80e19e466d7cc9591dfb7/fiftyone/core/fields.py#L1033) might be cleaner here

Reply:
good point. I'll double check in quick followup PR